CleanPy/replace_na.py

```diff
@@ -1,34 +1,60 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov 22 10:02:49 2018
 
-def replace_na(data, columns, replace="average", remove=False):
+@author: phuntsoktseten
+"""
+import pandas as pd
+import numpy as np 
+
+def replace_na(data, columns, replace="mean", remove=False):
     """
-    This function replaces NA values with either the min, max, median or average 
+    This function replaces NA values with either the min, max, median or mean
     value or removes the rows.
-
     Parameters
     ----------
     data : dataframe
         This is the dataframe that the function will use to replace NAs.
         
+    columns : list
+        List of columns to replace missing values on.
+        
     replace : string
         Specifies how to replace missing values.
-        values include: "min", "max", "median", "average"
+        values include: "mean","min", "max", "median"
     
     remove : boolean
         Tells the function whether or not to remove rows with NA.
         If True, replace argument will not be used.
-    
-    columns : list
-        List of columns to replace missing values on.
         
     Returns
     -------
     dataframe
-        A pandas dataframe where each NAs will be replaced by either 
-        min, max, median or average (specified by the user)
+        A pandas dataframe where each NAs will be replaced by either mean,
+        min, max, median  (specified by the user)
     
     >>> replace_na(pd.DataFrame(np.array([[0, 1], [NA, 1]])), replace="min", columns=[0])
     pd.DataFrame(np.array([[0, 1], [0, 1]]))
     """
-    
-    return
\ No newline at end of file
+
```

Comment:
should add code to pull column data based on the user-specified index(es).
column_names = list(data.columns.values)
    for column in column_names:

      if replace =="mean":

          for column_ind in column_names:
             mean= data[column_ind].mean()

             data.fillna({column_ind: mean})  ...

Reply:
oh wait - we should use the list of specified columns.. so it would look more like

```suggestion
column_names=list(data.colums.values)
if replace =="mean":
for column_ind in list_of_ind:
mean= data[column_ind].mean 
data.fillna({column_ind: mean})
```